### PR TITLE
Create Litmus-Leader.md

### PR DESCRIPTION
**Pull request by prithvi1307**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by ajeshbaby — APPROVED** [review]

File: Litmus-Leader.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Litmus Leader Program
+The Litmus Leader Program aims to identify community contributors & leaders in the Litmus community and reward them for their valuable contribution to Cloud-Native world by contributing to the LitmusChaos project.
+This program is the a Rewards and Recognitions campaign for the LitmusChaos project to award our most prominent contributors in various fields like development, advocacy, writing, content creation, etc. and be an example for others.
```
Comment (ksatchit) [inline Litmus-Leader.md:3]:
```suggestion
This program is a rewards and recognition campaign for the LitmusChaos project to award our most prominent contributors in various fields like development, advocacy, writing, content creation, etc.,
```

File: Litmus-Leader.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+----------------------
+
+## Leadway to Litmus Leader 💥:
+
+1. Active Participation: We encourage you to be participative in feedback and attending our Monthly Community Meetups, GitHub LitmusChaos discussions, and on other handles as well (Twitter, LinkedIn, Slack, etc.)
```
Comment (ksatchit) [inline Litmus-Leader.md:9]: Can we separate these points? Feedback on slack/github and attendance on community sync-ups is deemed enough for active project participation IMO. Participation on social media in more of advocacy.

File: Litmus-Leader.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+2. Active Advocacy: Talk/blog about your work/contributions/experience at LitmusChaos openly on your social media handles. Bring in referrals to expand our network.
```
Comment (ksatchit) [inline Litmus-Leader.md:11]:
```suggestion
2. Active Advocacy: Talk/blog about your work/contributions/experience at LitmusChaos openly on your social media handles. 
```

Advocacy is expected to organically cause adoption. Not sure if folks need to bring in referrals?

File: Litmus-Leader.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+3. Active Leadership: Apply for the higher leadership roles that LitmusChaos has to offer and be responsible for leading a team.
```
Comment (ksatchit) [inline Litmus-Leader.md:13]: 👍

File: Litmus-Leader.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+4. Active Productiveness: No. of pull requests in a time period. Show excellence!
```
Comment (ksatchit) [inline Litmus-Leader.md:15]: Could we add code reviews here as well?

**Pull request closed without merging**